packages/calcite-components/src/components/chip/chip.scss

```diff
@@ -1,73 +1,86 @@
-:host([scale="s"]) {
-  @apply text-n2 h-6;
-  --calcite-chip-spacing-s-internal: theme("spacing[1]");
-  --calcite-chip-spacing-l-internal: theme("spacing[2]");
-  .close,
-  .select-icon--active {
-    @apply h-4 w-4;
-  }
+/**
+ * CSS Custom Properties
+ *
+ * These properties can be overridden using the component's tag as selector.
+ *
+ * @prop --calcite-chip-background-color: Specifies the background color of the component.
+ * @prop --calcite-chip-border-color: Specifies the border color of the component.
+ * @prop --calcite-chip-box-shadow: Specifies the box shadow of the component.
+ * @prop --calcite-chip-corner-radius: Specifies the corner radius of the component.
+ * @prop --calcite-chip-icon-color: Specifies the color of the component's icon property.
+ * @prop --calcite-chip-text-color: Specifies the color of the component's `"default"` slot.
+ * @prop --calcite-chip-close-icon-color: Specifies the color of the component's `closable` icon.
+ * @prop --calcite-chip-close-background-color: Specifies the color of the component's `closable` background color.
+ * @prop --calcite-chip-close-background-color-hover: Specifies the hover color of the component's `closable` background color.
```

Comment:
Replace state specific tokens with a token reassignment in the `:host(:[state])`

Reply:
I think that would prevent overrides from being possible.

This allows for "when the close button is hovered, change the close button background color", not "when the host element is hovered, change the close button background color"

Reply:
Applies to other component with similar internal div as well.

Reply:
It is still possible it just requires people writing overrides to add the override to each state as well. I went over the pros and cons of this with Franco and he agreed.

Reply:
Sorry I forgot about internal component state.